.gitpod.Dockerfile

```
@@ -1,4 +1,4 @@
-FROM blockstack/stacks-blockchain:2.05.0.0.0-stretch as corenode
+FROM blockstack/stacks-blockchain:2.05.0.3.0-stretch as corenode
```

Upgrading version once more. Is `stretch` still needed? This is becoming an outdated debian distro, and the alpine image is much lighter.

```suggestion
FROM blockstack/stacks-blockchain:2.05.0.4.0 as corenode
```

---

If it's OK with you, I'll open this one as an issue for v7... I'm not sure how this could affect our test suites